src/Card/Card.levers.js

```diff
@@ -7,4 +7,23 @@ export const cardAlignOptions = ["left", "center", "right"];
 /**
  * Available HTML elements of the container.
  */
-export const cardElements = ["div", "span", "article", "li"];
```

Comment:
@rebeccahongsf I'm removing "span" and "li" since "span" is a bit more suitable for inline/1-liner text, and "li" could be a bit dangerous to use on its own as the outer wrapper. I appreciate your consideration for accessibility (which is probably why you added the "li" option), but it's a bit safer if they could wrap the whole card within an `<li>` instead. Also, these 2 elements can introduce some unwanted base styles into the component 😄

Reply:
I am ok with `span` going away but Caryl just had a developer session on having cards in a list as `li` elements. I think we should maintain that one.

Reply:
Huh...using `li` as the direct outer wrapper? Hmmm....ok. Let's keep it then. I think it does introduce some style changes. Let me double check.

Reply:
Added `li` back, and line height class so there will be no style change when using `li`